src/hooks/product-hook.js

```diff
@@ -0,0 +1,47 @@
+import { useDispatch, useSelector } from 'react-redux';
```

Comment:
`use-Product-action` będzie lepszą nazwą pliku, bo to faktycznie tutaj się znajduje